api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoTrimestralCohortQueries.java

```diff
@@ -44,14 +44,15 @@ public CohortDefinition getA() {
     CohortDefinition transferredIn =
         getNumberOfPatientsTransferredInFromOtherHealthFacilitiesDuringCurrentMonth();
     CompositionCohortDefinition cd = new CompositionCohortDefinition();
-    cd.addParameters(getParameters());
-
-    String yearBefore =
-        "onOrAfter=${onOrAfter-12m},onOrBefore=${onOrBefore-12m},location=${location}";
-    cd.addSearch("startedArt", map(startedArt, yearBefore));
-
-    String less12m = "onOrAfter=${onOrAfter-12m},onOrBefore=${onOrBefore},location=${location}";
-    cd.addSearch("transferredIn", map(transferredIn, less12m));
+    cd.addParameter(new Parameter("onOrAfter", "onOrAfter", Date.class));
+    cd.addParameter(new Parameter("onOrBefore", "onOrBefore", Date.class));
+    cd.addParameter(new Parameter("location", "location", Location.class));
+    cd.addSearch("startedArt", mapStraightThrough(startedArt));
+    cd.addSearch(
+        "transferredIn",
+        map(
+            transferredIn,
+            "onOrAfter=${onOrAfter},onOrBefore=${onOrBefore+12m},location=${location}"));
```

Comment:
So you are looking into 1 year a head of the end of reporting period?

Reply:
onOrBefore=${onOrBefore+12m}

Reply:
This is pertaining the changes Yassin is doing to the cohort report period, i merged develop into this PR so it was included. We will need to reachout to him for further clarifications

Reply:
Ok, that is clear

Reply:
@amosh101 @ningosi  Guys this commit is reverting the changes done in #45. Please undo.

Reply:
Undone, please confirm.

Reply:
@ynurmahomed Please refer to https://github.com/jembi/openmrs-module-eptsreports/pull/47/files#r413873192,